include/script_api/macros.h

```diff
@@ -568,6 +568,11 @@
 /// The given arguments can be accessed from the API function using `thread->ptrReadPos`.
 #define EVT_CALL(FUNC, ARGS...)                     EVT_CMD(EVT_OP_CALL, (Bytecode) FUNC, ##ARGS),
 
+/// Unknown cmd 90
+#define EVT_90(UNK)                             EVT_CMD(EVT_OP_90, UNK),
+
+/// Prints debug message
+#define EVT_DEBUG_PRINT(UNK)                    EVT_CMD(EVT_OP_DEBUG_PRINT, UNK),
```

Comment:
We can probably call these:
EVT_90 --> EVT_DEBUG_PRINT_STR or EVT_DEBUG_LOG, bases on usage, obviously its not implemented in the release build (see func_802C6E14).
EVT_DEBUG_PRINT --> EVT_DEBUG_PRINT_VAR